modules/ducktests/tests/ignitetest/tests/auth_test.py

```diff
@@ -0,0 +1,147 @@
+from ignitetest.utils.version import DEV_BRANCH, LATEST, IgniteVersion
+
+WRONG_PASSWORD = "wrong_password"
+TEST_USERNAME = "admin"
+TEST_PASSWORD = "qwe123"
+TEST_PASSWORD2 = "123qwe"
+
+ADD_USER = 'adduser'
+UPDATE_USER = 'updateuser'
+REMOVE_USER = 'removeuser'
+
+
+# pylint: disable=W0223
+class AuthenticationTests(IgniteTest):
+    """
+    Tests Ignite Control Utility Activation command with enabled Authentication
+    """
+    NUM_NODES = 2
+
+    @cluster(num_nodes=NUM_NODES - 1)
+    @ignite_versions(str(DEV_BRANCH), str(LATEST))
+    def test_activate_with_authentication(self, ignite_version):
+        """
+        Test activate cluster.
+        Authentication enabled
+        """
+
+        config = IgniteConfiguration(
+            cluster_state="INACTIVE",
+            auth_enabled=True,
+            version=IgniteVersion(ignite_version),
+            data_storage=DataStorageConfiguration(
+                default=DataRegionConfiguration(persistent=True)
+            )
+        )
+
+        servers = IgniteService(self.test_context, config=config, num_nodes=self.NUM_NODES - 1)
+
+        servers.start()
+
+        ControlUtility(cluster=servers, username=DEFAULT_AUTH_USERNAME, password=DEFAULT_AUTH_PASSWORD).activate()
+
+        # Run command with right password
+        check_authenticate(servers, DEFAULT_AUTH_USERNAME, DEFAULT_AUTH_PASSWORD)
+
+        # Run command with wrong password
+        check_authenticate(servers, DEFAULT_AUTH_USERNAME, WRONG_PASSWORD, True)
+
+    @cluster(num_nodes=NUM_NODES)
+    @ignite_versions(str(DEV_BRANCH))
+    def test_change_users(self, ignite_version):
+        """
+        Test add, update and remove user
+        """
+        config = IgniteConfiguration(
+            cluster_state="INACTIVE",
+            auth_enabled=True,
+            version=IgniteVersion(ignite_version),
+            data_storage=DataStorageConfiguration(
+                default=DataRegionConfiguration(persistent=True),
+            )
+        )
+
+        servers = IgniteService(self.test_context, config=config, num_nodes=self.NUM_NODES - 1)
+
+        servers.start()
+
+        ControlUtility(cluster=servers, username=DEFAULT_AUTH_USERNAME, password=DEFAULT_AUTH_PASSWORD).activate()
+
+        client_cfg = config._replace(client_mode=True, discovery_spi=from_ignite_cluster(servers))
+
+        # Add new user
+        check_authenticate(servers, TEST_USERNAME, TEST_PASSWORD, True)
+        self.run_with_creds(client_cfg, ADD_USER, TEST_USERNAME, TEST_PASSWORD)
+        check_authenticate(servers, TEST_USERNAME, TEST_PASSWORD)
+
+        # Update user password
+        self.run_with_creds(client_cfg, UPDATE_USER, TEST_USERNAME, TEST_PASSWORD2)
+        check_authenticate(servers, TEST_USERNAME, TEST_PASSWORD2)
+
+        # Remove user
+        self.run_with_creds(client_cfg, REMOVE_USER, TEST_USERNAME, free=False)
+        check_authenticate(servers, TEST_USERNAME, TEST_PASSWORD, True)
```

Comment:
Please, add negative tests - check that command wouldn't be executed with the wrong
* default user password
* test user password
* wrong username